app/views/api/v3/analytics/user_analytics/_progress_hypertension_report.html.erb

```diff
@@ -14,7 +14,7 @@
         <%= t("progress_tab.hypertension_report_title") %> report
       </h1>
       <p class="m-0px p-0px ta-left fw-normal fs-16px c-grey-dark">
-        <%= t("progress_tab.last_updated_at", date: DateTime.parse(@user_analytics.last_updated_at).to_date.strftime("%d-%b-%y at %I:%M %P")) %>
+        <%= t("progress_tab.last_updated_at", date: DateTime.parse(@user_analytics.last_updated_at).to_date.strftime(@date_format+ " at %I:%M %P")) %>
```

Comment:
If we've got `@date_format`, should just have a `@time_format` too? And if they're never used separately, maybe just an `@timestamp_format` or something?

Reply:
Can also define `DateTime.parse(@user_analytics.last_updated_at).to_date.strftime(@date_format+ " at %I:%M %P")` in a method in the controller since it's being used in multiple places.

Reply:
Moved date parsing logic to a function in ApplicationHelper. Also added a time format in `config/initializers/time_and_date_formats`